src/devices/Bmxx80/Bmx280Base.cs

```diff
@@ -169,6 +170,7 @@ public override bool TryReadPressure(out Pressure pressure)
         /// Contains <see cref="double.NaN"/> otherwise.
         /// </param>
         /// <returns><code>true</code> if pressure measurement was not skipped, otherwise <code>false</code>.</returns>
+        [ObsoleteAttribute("TryReadAltitude is obsolete. Please use Iot.Device.Common.WeatherHelper instead.")]
```

Comment:
Same comment goes to here and all other of your Obsolete Attributes. The main idea of the device bindings is to make the api surface easy to use, and using the WeatherHelper requires some deeper knowledge and some extra codes from the consuming side in order to get right. Plus it looks like WeatherHelper doesn't have TryRead equivalents which you would want in case you wanted to obsolete this API. Same comment going forward, it's fine to change our inner implementation to use WeatherHelper, but don't obsolete the current API, and keep the samples using the device bindings' API.

Reply:
Done (pending commit)